agriculture/demo/website_page.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="website_page_6" model="website.page">
+        <field name="website_id" ref="website.default_website"/>
+        <field name="url">/pricing</field>
+        <field name="view_id" ref="pricing"/>
+        <field name="is_published" eval="True"/>
+    </record>
+    <record id="website_page_8" model="website.page">
+        <field name="website_id" ref="website.default_website"/>
+        <field name="url">/red-revolution-tomato-seeds</field>
+        <field name="view_id" ref="red_revolution_tomato_seeds"/>
+        <field name="is_published" eval="True"/>
+    </record>
+    <record id="website_page_9" model="website.page">
+        <field name="website_id" ref="website.default_website"/>
+        <field name="url">/red-revolution-tomato-seeds-1</field>
+        <field name="view_id" ref="red_revolution_tomato_seeds_1"/>
+        <field name="is_published" eval="True"/>
+    </record>
```

Comment:
Where this page is used?

Reply:
![agriculture](https://github.com/odoo/industry/assets/120459793/2f04510d-297f-48d5-82a7-a79f72a8c3c2)


So this page is shown in product stories dropdown menu